docker/docker-dev-direct-single-tenant/datasources/mt-graphite-api

```diff
@@ -1,7 +0,0 @@
-{
-  "name":"mt-graphite-api",
-  "type":"graphite",
-  "url":"http://localhost:8080",
-  "access":"direct",
-  "isDefault":true
-}
```

Comment:
agreed. I ensured all envs have a graphite and metrictank datasource, with the metrictank one set as default.